repositories/byod/_posts/2016-12-13-using-your-own-dockerfiles.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Using Your Own Dockerfiles
+short: Using Your Own Dockerfiles
+category: repositories
+order: 6
+---
+
+Dockerfiles are often used for managing configuration and automating the build process for a service. If your repositories have Dockerfiles, it’s easy to get them set up in Runnable. At the end of this guide, you'll have a running container for a repository, built from its Dockerfile.
+
+*Note: Using your own Dockerfiles is a great way to start using Runnable, but some features are reserved for apps that use Docker Compose:*
```

Comment:
Yeah good catch (about linking to the Compose guide). Will you do that in your branch?